crates/host/src/wasmbus/mod.rs

```diff
@@ -1101,12 +1179,70 @@ impl Lattice {
                 instances: HashMap::default(),
             });
         if let hash_map::Entry::Vacant(entry) = instances.entry(link_name.clone()) {
-            debug!(?path, "spawn provider process");
-            let child = process::Command::new(&path)
+            let id = Ulid::new();
+            let async_nats::ServerInfo { host, port, .. } = self.nats.server_info();
+            let invocation_seed = self
+                .cluster_key
+                .seed()
+                .context("cluster key seed missing")?;
+            let links = self.links.read().await;
+            let link_definitions: Vec<_> = links
+                .values()
+                .filter(|ld| ld.provider_id == claims.subject && ld.link_name == link_name)
+                .collect();
+            let data = serde_json::to_vec(&json!({
```

Comment:
Naming nit: `host_data`, which should also be a struct type (defined in `provider-sdk/src/core.rs` for reference)